### DISCUSSION TITLE
Which control can be used to presenting log? and how?

### DISCUSSION THREAD

**Discussion by herolss** [microsoft/WinUI-Gallery#1039] [Q&A]
i use texblock to present communicating messages log. its so ugly
does winui-gallery delelop a logger control?

**Comment by marcelwgn** [comment]
The WinUI Gallery is only a sample gallery showcasing WinUI 3 controls and APIs. Regarding rendering a log, unfortunately there is no dedicated control for this (what exactly differs a log from regular text?). Maybe the RichTextBlock control might be helpful for this since it supports RTF.
